.github/ISSUE_TEMPLATE/enhancement-refinement-template.md

```diff
@@ -0,0 +1,28 @@
+# Acceptance criteria
+
+## Base requirements
+
+- [ ] User stories describing what new component is meant to achieve
```

Comment:
I would prefer we document this as a description rather than a line item. But if the ideas is just ensuring that an existing enhancement includes these details, it might be fine as a line item?